HeavyIonsAnalysis/JetAnalysis/python/candidateBtaggingMiniAOD_cff.py

```diff
@@ -1,37 +1,101 @@
 import FWCore.ParameterSet.Config as cms
 
 from RecoBTag.ImpactParameter.pfImpactParameterTagInfos_cfi import pfImpactParameterTagInfos
-pfImpactParameterTagInfos.jets = "slimmedJets"
+pfImpactParameterTagInfos.jets = "akCs0PFpatJets"
 pfImpactParameterTagInfos.candidates = "packedPFCandidates"
 pfImpactParameterTagInfos.primaryVertex = "offlineSlimmedPrimaryVertices"
 from RecoBTag.SecondaryVertex.pfSecondaryVertexTagInfos_cfi import pfSecondaryVertexTagInfos
+
 # leave IVF workflow commented out for reference -matt
```

Comment:
Obsolete comment. With this PR, the workflow is now active. This should be fixed.